src/pages/LandingPage.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios"
+import { useState, useEffect } from "react"
+import { IcOutlineThumbUp,IcRoundThumbUp, IcOutlineWatchLater, IcRoundWatchLater } from '../assets/index'
+import { useWatchLaterContext } from "../contexts/watchlater-context"
+import { useLikedVideosContext } from "../contexts/liked-videos-context.js"
```

Comment:
suggestion: create an index file in the contexts folder and export all contexts from it